src/wallet3/db_schema.cpp

```diff
@@ -26,6 +26,7 @@ namespace wallet
             id INTEGER PRIMARY KEY AUTOINCREMENT,
             block INTEGER NOT NULL REFERENCES blocks(height) ON DELETE CASCADE,
             hash TEXT NOT NULL,
+            public_key TEXT NOT NULL,
```

Comment:
tx can have more than one pubkey, so better to put this as either `tx_public_key` or `key_derivation` on outputs table.